docs/src/pages/nextjs/appdir.mdx

```diff
@@ -24,10 +24,8 @@ const auth = (req: Request) => ({ id: "fakeId" }); // Fake auth function
 // FileRouter for your app, can contain multiple FileRoutes
 export const ourFileRouter = {
   // Define as many FileRoutes as you like, each with a unique routeSlug
-  imageUploader: f
+  imageUploader: f({ image: { maxFileSize: "4MB" } })
     // Set permissions and file types for this FileRoute
```

Comment:
comments should probably be moved above the route def.

Reply:
Will do in docs pass, don't want more conflicts